File: acceptance/71-simplify path.js
var simplifyPath = function (path) {
    let finalPath = [];
    let Spath = path.split('/');
    Spath.forEach((element) => {
        if (element != '' && element != '.' && element != '..') {
            finalPath.push(element);
        }
        else if (element == '..') {
            if (finalPath.length > 0)
                finalPath.pop();
        }
    })
    return '/' + finalPath.join('');
}

File: acceptance/sauravkhanal/71-Simplify-Path.py
class Solution(object):
    def simplifyPath(self, path):
        """
        :type path: str
        :rtype: str
        """
        path_stack = []
        for s in path.split("/"):
            if not s:
                continue
            elif s == ".":
                continue
            elif s == "..":
                if path_stack:
                    path_stack.pop()
            else:
                path_stack.append(s)
        return "/" + "/".join(path_stack)